apps/CEventService.API/Data/DataSeeder.cs

```diff
@@ -0,0 +1,129 @@
+using CEventService.API.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace CEventService.API.Data
+{
+    public static class DataSeeder
```

Comment:
This dataSetter looks good to me but I think it could be better if you run a .sql script directly, it's faster and also it's neater when we want to add more configuration to the initial script, your data setter could run that script at startup.

Reply:
This seeder was an extra effort made by me. In order to make things easier when testing. This was not planned on the User story acceptance criteria. As your coment is "personal". I'd rather work on much important matter such as the user authentication. @CatCoder36

Reply:
In case of not taking into account the scalability of the code and good practices I suggest deleting or refactoring the “extra effort” as it is considered unnecessary code, when making code in the repository this code ceases to be “personal” because it is part of everyone's work.

Reply:
As the code does not match your expectation of scalability and you are the reviewer. I'll delete the dataseeder in the next commit. :+1: